README.md

```diff
@@ -111,6 +111,66 @@ When running on cluster node, you will also need to distribute this keytab, belo
 
 When Spark application is started, it will transparently track the execution plan of submitted SQL/DF transformations, parse the plan and create related entities in Atlas.
 
+Spark models vs Hive models
+====
+
+SAC classifies table related entities with two different kind of models: Spark / Hive.
```

Comment:
I am not sure classifies conveys the right meaning since SAC does not create Hive models.
Maybe "SAC creates Spark models when spark manages the catalog and just references Hive meta store created models (Hive models) when the catalog is managed by Hive meta store"

Reply:
> SAC creates Spark models when spark manages the catalog and just references Hive meta store created models (Hive models) when the catalog is managed by Hive meta store

This is not true - we still deal with table entities with Spark models if we leverage embedded hive metastore or so. That's why explaining this is not going to be intuitive.